Update script

BerglindReynis opened the pull request:
Updating createdatabase.sql

BerglindReynis committed:
Update script

Updating createdatabase.sql

eythort-mm commented on guides/implementation/central_mssql.rst:
```diff
@@ -99,4 +93,4 @@ Verify the service is running smoothly:
   systemctl status mmcentral
 
-If there are any database connection issues, the service will fail to start. Check the Central startup log for errors, located in the Central data directory at either ``/var/mmsuite/mmcentral/logs`` or ``C:\ProgramData\Men and Mice\Central\logs``.
+If there are any database connection issues, the service will fail to start. Check the Central startup log for errors, located in the Central data directory at either ``/var/micetrosuite/mmcentral/logs`` or ``C:\ProgramData\Micetro\Central\logs``.
```
file path locations are still /var/mmsuite and C:\ProgramData\Men and Mice\ 

But the CreateDatabase script is being updated in the DevOps PR here https://dev.azure.com/menandmice/_git/Suite/pullrequest/9761, where I decided to use the opportunity and change the name of the database objects we create to micetro. Changing the filepaths on Micetro servers is whole another (and bigger) story

eythort-mm committed:
Update CreateDatabase.sql

Duplicate from changes being reviewed here
https://dev.azure.com/menandmice/_git/Suite/pullrequest/9761

eythort-mm commented on scripts/CreateDatabase.sql:
I copied the content from the devops PR here https://dev.azure.com/menandmice/_git/Suite/pullrequest/9761

It's a bit uncertain where the authority for this script is, it was duplicated in two folders in the Suite repo, and also apparently in this repo

eythort-mm commented on guides/implementation/central_mssql.rst:
```diff
@@ -19,16 +19,10 @@ Micetro requires the creation of a new database on your server with the ``Latin1
 
 The `CreateDatabase.sql <https://github.com/menandmice/micetro_docs/blob/latest/scripts/CreateDatabase.sql>`_ is recommended for setting up the database and server. However, it also includes configurations for the SQL server that may not be needed for desired. Work with your DBA to review the script and execute only the necessary sections. Adjust the path strings (default is ``C:/Data``) and upda ...
```
```suggestion
The `CreateDatabase.sql <https://github.com/menandmice/micetro_docs/blob/latest/scripts/CreateDatabase.sql>`_  can be used for setting up a blank database for Micetro with the recommended settings. By default, the script also sets up a user account, micetroDBUser, whose password must be adjusted before it is run. 
```

eythort-mm commented on guides/implementation/central_mssql.rst:
```diff
@@ -19,16 +19,10 @@ Micetro requires the creation of a new database on your server with the ``Latin1
-* The establishment of an empty database named ``mmsuite`` with the ``COLLATE SQL_Latin1_General_CP1_CS_AS`` collation.
+* The creation of a user named ``micetroDBUser`` with a given password.
+* The establishment of an empty database named ``micetro`` with the ``Latin1_General_100_CS_AS_KS_WS_SC_UTF8`` collation.
 
```
Perhaps add a note on the recovery mode. Something like
Note that the recovery model of the created database is by default Simple. If you wish to use a Full recovery model, a transaction log backup job is required.

eythort-mm commented on guides/implementation/central_mssql.rst:
I would suggest to change the note "It is recommended to a have a dedicated database administrator ...." 
to 
Micetro does not automatically perform index maintentance or database backups for the Micetro database. It is recommended to have a database administrator set up a scheduled job to maintain indexes as well as take periodic backups of the database and transaction logs (if using Full recovery model)

eythort-mm commented on guides/implementation/central_mssql.rst:
I would change this 
Micetro requires the creation of a new database on your server with the ``Latin1_General_100_CS_AS_KS_WS_SC_UTF8`` collation. Make sure there is a login (either Windows or SQL server authenticated) with ``db_owner`` privileges for this database and set the default schema set to ``mmCentral``.

to 

Micetro requires the creation of a new database on your server with a case sensitive and accent sensitive collation. For SQL Server 2019 and newer, we recommend to use a UTF8 collation such as ``Latin1_General_100_CS_AS_KS_WS_SC_UTF8``.  For older versions of SQL server, or Micetro versions 11.0 and older, we recommend the ``SQL_Latin1_General_CP1_CS_AS`` collation.

BerglindReynis committed:
Update central_mssql.rst

Correcting path to startup log

BerglindReynis committed:
Update guides/implementation/central_mssql.rst

Co-authored-by: Eythor <48280310+eythort-mm@users.noreply.github.com>

BerglindReynis committed:
Update central_mssql.rst

Updated according to review

eythort-mm commented on guides/implementation/central_mssql.rst:
```diff
@@ -10,25 +10,22 @@ Microsoft SQL Server
   It is recommended to keep the network latency between the SQL Server and Micetro Central **below 5 milliseconds**. Latency above this level can lead to performance issues.
 
-  It is recommended to have a dedicated database administrator (DBA) to manage and maintain the SQL Server database.
+  Micetro does not automatically perform index maintentance or database backups. It is recommended to have a database administrator set up a scheduled task to maintain indexes and regularly back up the database and transaction logs, (when using teh Full recovery model).
```
teh to the ;)

eythort-mm commented on guides/implementation/central_mssql.rst:
```diff
@@ -73,7 +70,7 @@ For Micetro Central on Windows, you have two authentication methods to choose fr
 For Micetro Central on Linux
 ******************************
-Navigate to the data directory (usually located at ``/var/mmsuite/mmcentral``) and edit the ``preferences.cfg`` file as follows:
+Navigate to the data directory (usually located at ``/var/micetro/mmcentral``) and edit the ``preferences.cfg`` file as follows:
```
this is still `/var/mmsuite/mmcentral`

BerglindReynis committed:
Minor text changes

Minor corrections
